includes/settings/settings.php

```diff
@@ -252,6 +252,13 @@ protected function create_tabs() {
 				'label' => esc_html__( 'General', 'elementor' ),
 				'sections' => [
 					'general' => [
+						'label' => esc_html__( 'General', 'elementor' ),
+						'callback' => function() {
+							printf(
+								'<p>%s</p><br><hr><br>',
```

Comment:
```suggestion
								'<p>%s</p><br/><hr/><br/>',
```

Reply:
Not sure about this. This is an XHTML convention. HTML5 allows self-closing tags.

Read this reference: https://developer.mozilla.org/en-US/docs/Glossary/Void_element

> Self-closing tags (`<tag />`) do not exist in HTML.
>
> If a trailing `/` (slash) character is present in the start tag of an HTML element, HTML parsers ignore that slash character.